### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by MarrekNozka**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @Koles359

### CODE REVIEW

**Comment by Koles359** [comment]
Projekt jsem vypracoval sám s tím, že jsem si procvičil práci s moduly a naučil se pracovat s ASCII tabulkou. Myslím, že si zasloužím jedničku, protože jsem nad tím strávil hodně času a dal jsem si s tím záležet.

File: cesarova_sifra.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+
+import modul_k_cesarove_sifre
+
+start = input("Chcete spustit program? y/n : ")
```
Comment (MarrekNozka) [inline cesarova_sifra.py:6]: Co sem třeba napsat `start ='y'`. To je přece jesné, že ho chci spustit. :-)

File: cesarova_sifra.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    else:
+        if vstup == 1:
+            zadani = input("Zadej zprávu k šifrování (bez interpunkce, diakritiky a mezer!!!): ").lower()
+            posun = int(input("Zadej posun: "))
+            print("\n")
+            print(modul_k_cesarove_sifre.sifrovani(zadani, posun))
+            print("\n")
+            start = input("Chcete v programu pokračovat? y/n : ")
+        else:
+            zprava = input("Zadej zprávu k dešifrování (bez interpunkce, diakritiky a mezer!!!): ").lower()
```
Comment (MarrekNozka) [inline cesarova_sifra.py:27]: Tady někdy by to chtělo otevřít soubor a načíst zprávu z toho souboru. Nebudete to přece zadávat 30 dokolečka stále znovu to stejné.

**Review by MarrekNozka — COMMENTED** [review]
Ano. Je vidět, že jste si pohrál. Škoda, že tam není práce se souborem. Ale uznávám, že ze zadání to není příliš jasné.
:one:

**Pull request closed without merging**